toolbox/load_agol_layer/src/utils.js

```diff
@@ -164,7 +164,7 @@ const getEsriJson = async (url) => {
  */
 const getTruncateMutation = (tableName) => `
   mutation Delete${tableName} {
-    delete_${tableName}(where: {}) {
+    delete_geo_${tableName}(where: {}) {
```

Comment:
Pretty cool that Hasura includes the schema in these generated mutations. I hadn't caught that detail before now.